src/core/lib/iomgr/workqueue.h

```diff
@@ -49,34 +50,25 @@
 
 /* grpc_workqueue is forward declared in exec_ctx.h */
 
```

Comment:
Documented. enqueue is a little loose: I'm planning on changing the semantics a little in #7279, so I documented the union of that - it'll give us some future flexibility also.